src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -455,11 +461,21 @@ const applyStyles = (document, body) => {
 	}
 
 	if (body) {
-		body.classList.remove('vscode-light', 'vscode-dark', 'vscode-high-contrast');
+		body.classList.remove('vscode-light', 'vscode-dark', 'vscode-high-contrast', 'reduce-motion', 'enable-motion', 'using-screen-reader');
 		if (initData.activeTheme) {
 			body.classList.add(initData.activeTheme);
 		}
 
+		if (initData.reduceMotion) {
+			body.classList.add('reduce-motion');
+		} else {
+			body.classList.add('enable-motion');
```

Comment:
Do we need `enable-motion` since it's the default? The other classes here only get added if they are not the default value

Reply:
removed, we use in on core as it makes some css easier but not critical

Reply:
removed, we use in on core as it makes some css easier but not critical